### ISSUE TITLE
Bug in the Neo4j Streaming Data Integrations User Guide

### ISSUE THREAD

**Issue by davidoliverSP2** [neo4j-contrib/neo4j-streams#279]
https://neo4j.com/docs/labs/neo4j-streams/current/docker/
 
version: '3'

services:

neo4j:

```
    image: neo4j:3.4

    hostname: neo4j

    container_name: neo4j

    ports:

    - "7474:7474"

    - "7687:7687"

    depends_on:

    - kafka

    volumes:

    - ./neo4j/plugins:/plugins:/plugins   <= /plugins: repeated twice

    environment:

      NEO4J_AUTH: neo4j/streams

      NEO4J_dbms_logs_debug_level: DEBUG

      # KAFKA related configuration

      NEO4J_kafka_zookeeper_connect: zookeeper:12181

      NEO4J_kafka_bootstrap_servers: kafka:19092
```

zookeeper:

```
    image: confluentinc/cp-zookeeper:latest

    hostname: zookeeper

    container_name: zookeeper

    ports:

   - "12181:12181"

    environment:

      ZOOKEEPER_CLIENT_PORT: 12181
```

 kafka:

```
    image: confluentinc/cp-kafka:latest

    hostname: kafka

    container_name: kafka

    ports:

    - "19092:19092"

    depends_on:

    - zookeeper

    environment:

      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1

      KAFKA_ZOOKEEPER_CONNECT: zookeeper:12181

      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:19092
```

` /plugins:` is repeated twice

**Comment by mroiter-larus** [comment]
Hi @davidoliverSP2, i already fixed this typo and included it into the PR #276. It will be available soon.
However, thanks for pointing it out!

**Comment by conker84** [comment]
closed via 27cddd4
